packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/Contracts.jsx

```diff
@@ -95,11 +104,26 @@ const customHeaderPerDoctype = {
 }
 
 const DumbContracts = ({ contracts, account, konnector }) => {
-  const contractData = contracts.data ? contracts.data : contracts
   const { t } = useI18n()
+  const contactsAppQuery = useFetchJSON('GET', '/apps/contacts')
```

Comment:
Why using useFetchJSON and not an useQuery with this kind of definition? 
```
Q('io.cozy.apps').getById('io.cozy.apps/contacts')`
```

Reply:
Like that, we can make / use the query inside the components without having to pass down a props since the query will not be launched twice

Reply:
> Why using useFetchJSON and not an useQuery with this kind of definition?
> 
> ```
> Q('io.cozy.apps').getById('io.cozy.apps/contacts')`
> ```

Parce que retour de vacances ? :p